<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopterInterface</class>
 <widget class="QWidget" name="CopterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/models/Icons/Orthogonal View-96.png</normaloff>:/models/Icons/Orthogonal View-96.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
        <attribute name="toolTip">
         <string>Orientation</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="orientationLayout">
           <item>
            <spacer name="orientationSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QProgressBar" name="rollBar">
             <property name="minimum">
              <number>-180</number>
             </property>
             <property name="maximum">
              <number>180</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>Roll: %v°</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="pitchBar">
             <property name="minimum">
              <number>-180</number>
             </property>
             <property name="maximum">
              <number>180</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>Pitch %v°</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="yawBar">
             <property name="minimum">
              <number>-180</number>
             </property>
             <property name="maximum">
              <number>180</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>Yaw: %v°</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="speedBar">
             <property name="maximum">
              <number>120</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>Speed: %v km/h</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/models/Icons/Gyroscope-96.png</normaloff>:/models/Icons/Gyroscope-96.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
        <attribute name="toolTip">
         <string>Raw IMU data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="ImuPlot" name="imuPlot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/models/Icons/Console-96.png</normaloff>:/models/Icons/Console-96.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
        <attribute name="toolTip">
         <string>Terminal</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTextBrowser" name="terminalBrowser">
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLineEdit" name="terminalEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="terminalSendButton">
             <property name="toolTip">
              <string>Send command to car</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/models/Icons/Send File-96.png</normaloff>:/models/Icons/Send File-96.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="terminalClearButton">
             <property name="toolTip">
              <string>Clear terminal</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/models/Icons/Delete2-96.png</normaloff>:/models/Icons/Delete2-96.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/models/Icons/Magnet-96.png</normaloff>:/models/Icons/Magnet-96.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
        <attribute name="toolTip">
         <string>Calibration</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Magnetometer</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="MagCal" name="magCal" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/models/Icons/Satellite-96.png</normaloff>:/models/Icons/Satellite-96.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
        <attribute name="toolTip">
         <string>NMEA</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="NmeaWidget" name="nmeaWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/models/Icons/Settings-96.png</normaloff>:/models/Icons/Settings-96.png</iconset>
        </attribute>
        <attribute name="title">
         <string/>
        </attribute>
        <attribute name="toolTip">
         <string>Configuration</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Copter Parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ConfCommonWidget" name="confCommonWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="fwLabel">
         <property name="text">
          <string>FW x.y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="batteryBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>Battery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="clockLabel">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSpinBox" name="idBox">
       <property name="prefix">
        <string>Copter ID: </string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="pollBox">
       <property name="text">
        <string>Poll Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="joystickControlBox">
       <property name="text">
        <string>Joystick Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autopilotBox">
       <property name="text">
        <string>AutoPilot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="updateRouteFromMapBox">
       <property name="toolTip">
        <string>Upload route points as soon as they are created on the map</string>
       </property>
       <property name="text">
        <string>Route from map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearRouteButton">
       <property name="toolTip">
        <string>Clear stored route</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Delete2-96.png</normaloff>:/models/Icons/Delete2-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setClockButton">
       <property name="toolTip">
        <string>Set clock on STM32 board</string>
       </property>
       <property name="text">
        <string>STM</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Clock-96.png</normaloff>:/models/Icons/Clock-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setClockPiButton">
       <property name="toolTip">
        <string>Set clock on Raspberry Pi</string>
       </property>
       <property name="text">
        <string>Pi</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Clock-96.png</normaloff>:/models/Icons/Clock-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rebootPiButton">
       <property name="text">
        <string>Reboot PI</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Restart-96.png</normaloff>:/models/Icons/Restart-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shutdownPiButton">
       <property name="text">
        <string>Shutdown PI</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Shutdown-96.png</normaloff>:/models/Icons/Shutdown-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImuPlot</class>
   <extends>QWidget</extends>
   <header>imuplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NmeaWidget</class>
   <extends>QWidget</extends>
   <header>nmeawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MagCal</class>
   <extends>QWidget</extends>
   <header>magcal.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfCommonWidget</class>
   <extends>QWidget</extends>
   <header>confcommonwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>terminalEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>terminalSendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminalClearButton</sender>
   <signal>clicked()</signal>
   <receiver>terminalBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
